angular/base/tsconfig.json

```diff
@@ -16,9 +16,12 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2022",
+    "target": "es2015",
```

Comment:
Can we continue to target `es2022`? Or is there a reason we should not?

Reply:
Nope, we can keep it.